app/views/price_policies/index.html.haml

```diff
@@ -9,7 +9,7 @@
 
 %p= text("description")
 
-- if can?(:create, PricePolicy)
+- if can?(:create, PricePolicy) && !@product.skip_order_review?
   %ul.inline
     %li= link_to text("add"), [:new, current_facility, @product, :price_policy], class: "btn-add"
```

Comment:
```suggestion
- if can?(:create, PricePolicy) && !@product.skip_order_review?
  %ul.inline
    %li= link_to text("add"), [:new, current_facility, @product, :price_policy], class: "btn-add"
- elsif @product.skip_order_review?
  %ul.inline= text("skip_review_message", billing_mode: @product.billing_mode)
```
Message: "This product is set to Skip Review billing mode, price rules cannot be changed."